src/Impl/ConsumerPollingTrait.php

```diff
@@ -0,0 +1,72 @@
+<?php
+declare(strict_types=1);
+
+namespace Interop\Queue\Impl;
+
+use Interop\Queue\Message;
+
+/**
+ * The ConsumerPollingTrait is a common implementation of the polling algorithm for a Consumer
+ */
+trait ConsumerPollingTrait
+{
+    /**
+     * Polling interval in milliseconds
+     * @var integer
+     */
+    protected $pollingInterval = 1000;
+
+    /**
+     * Set polling interval in milliseconds.
+     */
+    public function setPollingInterval(int $msec): void
+    {
+        $this->pollingInterval = $msec;
```

Comment:
should return `$this` and return typehint `: self`

Reply:
@sylfabre ping

Reply:
@sylfabre ping

Reply:
@makasim forgot to push on Friday!

Reply:
@makasim forgot to push on Friday!

Reply:
@makasim forgot to push on Friday!

Reply:
@makasim forgot to push on Friday!

Reply:
@makasim forgot to push on Friday!

Reply:
@sylfabre

Reply:
@sylfabre ping